recipe/build.sh

```diff
@@ -12,11 +12,45 @@ then
 fi
 
 ./configure \
-        --prefix="${PREFIX}" \
-        --enable-shared \
-        --enable-pic \
-        --enable-libx264 \
-        --disable-doc \
-        --enable-gpl
+       --prefix="${PREFIX}" \
+       --disable-doc \
+       --disable-debug \
+       --disable-static \
+       --disable-stripping \
+       --enable-avisynth \
+       --enable-avresample \
+       --enable-fontconfig \
+       --enable-gpl \
+       --enable-ladspa \
+       --enable-libass \
+       --enable-libbluray \
+       --enable-libdcadec \
+       --enable-libfreetype \
+       --enable-libfribidi \
+       --enable-libgsm \
+       --enable-libmodplug \
+       --enable-libmp3lame \
+       --enable-libopencore_amrnb \
+       --enable-libopencore_amrwb \
+       --enable-libopenjpeg \
+       --enable-libopus \
+       --enable-libpulse \
+       --enable-libschroedinger \
+       --enable-libsoxr \
+       --enable-libspeex \
+       --enable-libssh \
+       --enable-libtheora \
+       --enable-libv4l2 \
+       --enable-libvidstab \
+       --enable-libvorbis \
+       --enable-libvpx \
+       --enable-libwebp \
+       --enable-libx264 \
+       --enable-libx265 \
+       --enable-libxvid \
+       --enable-pic \
+       --enable-shared \
+       --enable-version3 \
+       --enable-x11grab
```

Comment:
Why do we want this this? If we need it, we probably need install stuff with `yum` on the system.